src/noit_check_rest.c

```diff
@@ -101,6 +101,9 @@ add_metrics_to_node(stats_t *c, xmlNodePtr metrics, const char *type,
     xmlSetProp(tmp, (xmlChar *)"name", (xmlChar *)m->metric_name);
     buff[0] = m->metric_type; buff[1] = '\0';
     xmlSetProp(tmp, (xmlChar *)"type", (xmlChar *)buff);
+    if(!noit_apply_filterset(check->filterset, check, m)) {
+      xmlSetProp(tmp, (xmlChar *)"filtered", (xmlChar *)"true");
+    }
```

Comment:
I don't see any comment about this new filtered / _filtered attribute in the xml or json

Reply:
I noticed that the check pulling has no way to show which metrics are currently filtered out.  so I added an attribute.  I didn't add the "false" case b/c the documents are large and didn't need to get much larger.